Event/AcbFilePostValidate.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Sherlockode\AdvancedContentBundle\Event;
+
+use Symfony\Contracts\EventDispatcher\Event;
+
+class AcbFilePostValidate extends Event
+{
+    public const NAME = 'ACB_FILE.POST_VALIDATE';
```

Comment:
Lowercase is recommended for the event name: https://symfony.com/doc/current/components/event_dispatcher.html#naming-conventions